pdf/delivery-slip.product-tab.tpl

```diff
@@ -76,7 +76,7 @@
 						<td class="center"> &nbsp;</td>
 
 						<td>
-							{if isset($customization.datas[Product::CUSTOMIZE_TEXTFIELD]) && count($customization.datas[Product::CUSTOMIZE_TEXTFIELD]) > 0}
+							{if !empty($customization.datas[Product::CUSTOMIZE_TEXTFIELD])}
```

Comment:
I would add an `is_array` for checking that it is iterable.

Reply:
If defined, it's iterable. We are already in 2 foreach and they will crash before this line if we `$order_detail.customizedDatas` is null

Reply:
https://github.com/PrestaShop/PrestaShop/blob/develop/classes/Product.php#L5186